common/app/com/commercetools/sunrise/framework/localization/CountryFromSessionProvider.java

```diff
@@ -13,7 +14,7 @@
     private final CountryInSession countryInSession;
 
     @Inject
-    public CountryFromSessionProvider(final ProjectContext projectContext, final CountryInSession countryInSession) {
+    CountryFromSessionProvider(final ProjectContext projectContext, final CountryInSession countryInSession) {
```

Comment:
Nice that you removed the `public` modifier here. And we now could remove the `public` modifier from this class too.